src/components/Chip/stories.scss

```diff
@@ -0,0 +1,21 @@
+.story--components-buttontw--default {
```

Comment:
Can we add a strict rule for classNames? If it is a default setting, pls ignore this comment

Reply:
I'll try to add something when I configure prettrier eslint properly